pkg/statistics/handle/globalstats/global_stats_async.go

```diff
@@ -430,13 +430,15 @@ func (a *AsyncMergePartitionStats2GlobalStats) loadHistogramAndTopN(sctx session
 			hists = append(hists, h)
 			topn = append(topn, t)
 		}
-		select {
-		case a.histogramAndTopn <- mergeItem[*StatsWrapper]{
-			NewStatsWrapper(hists, topn), i,
-		}:
-		case <-a.cpuWorkerExitChan:
-			statslogutil.StatsLogger().Warn("ioWorker detects CPUWorker has exited")
-			return nil
+		if !(len(hists) == 0 && len(topn) == 0) {
+			select {
+			case a.histogramAndTopn <- mergeItem[*StatsWrapper]{
+				NewStatsWrapper(hists, topn), i,
+			}:
+			case <-a.cpuWorkerExitChan:
+				statslogutil.StatsLogger().Warn("ioWorker detects CPUWorker has exited")
+				return nil
+			}
```

Comment:
```suggestion
		// This only happens when there are no records for the histogram and TopN in the system. It may be caused by the DDL event not having been handled yet.
		if (len(hists) == 0 && len(topn) == 0) {
			continue
		}
		select {
		case a.histogramAndTopn <- mergeItem[*StatsWrapper]{
			NewStatsWrapper(hists, topn), i,
		}:
		case <-a.cpuWorkerExitChan:
			statslogutil.StatsLogger().Warn("ioWorker detects CPUWorker has exited")
			return nil
```